rxbinding/src/androidTest/java/com/jakewharton/rxbinding/view/RxViewAttachTest.java

```diff
@@ -0,0 +1,140 @@
+package com.jakewharton.rxbinding.view;
+
+import android.app.Instrumentation;
+import android.support.test.InstrumentationRegistry;
+import android.support.test.rule.ActivityTestRule;
+import android.support.test.runner.AndroidJUnit4;
+import android.view.View;
+import android.widget.FrameLayout;
+
+import com.jakewharton.rxbinding.RecordingObserver;
+
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import rx.Subscription;
+import rx.android.schedulers.AndroidSchedulers;
+
+import static com.google.common.truth.Truth.assertThat;
+import static com.jakewharton.rxbinding.view.ViewAttachEvent.Kind.ATTACH;
+import static com.jakewharton.rxbinding.view.ViewAttachEvent.Kind.DETACH;
+
+@RunWith(AndroidJUnit4.class)
+public final class RxViewAttachTest {
+  @Rule public final ActivityTestRule<RxViewAttachTestActivity> activityRule =
+      new ActivityTestRule<>(RxViewAttachTestActivity.class);
+
+  private final Instrumentation instrumentation = InstrumentationRegistry.getInstrumentation();
+  private FrameLayout parent;
+  private View child;
+
+  @Before public void setUp() {
+    RxViewAttachTestActivity activity = activityRule.getActivity();
+    parent = activity.parent;
+    child = activity.child;
+  }
+
+  @Test public void attaches() {
+    RecordingObserver<Object> o = new RecordingObserver<>();
+    Subscription subscription = RxView.attaches(child)
+        .subscribeOn(AndroidSchedulers.mainThread())
+        .subscribe(o);
+    o.assertNoMoreEvents(); // No initial value.
```

Comment:
In the test or the library itself? I thought about checking against it in the initial implementation of the Rxlifecycle PR, but `isAttachedToWindow()` is API 19+...